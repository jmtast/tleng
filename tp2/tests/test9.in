// Error de sem√°ntica: Pulso valor no igual a una figura
#tempo negra 120
#compas 2/5

const oct1 = 5;

voz (0)
{
  compas
  {
    silencio(blanca);
  }
}
