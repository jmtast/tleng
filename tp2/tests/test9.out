Error:
Pulso incorrecto. Debe ser el valor de una figura: 1, 2, 4, 8, 16, 32, 64. LÃ­nea 3
